app/controllers/censuses_controller.rb

```diff
@@ -5,7 +5,8 @@
 ###
 
 class CensusesController < ApplicationController
-  before_action :require_can_view_censuses!, except: [:date_range]
+  include WarehouseReportAuthorization
```

Comment:
Do we need a migration to grant people with this permission report access?

Reply:
That comes in the next PR https://github.com/greenriver/hmis-warehouse/pull/1316/files